tests/circomkit.test.ts

```diff
@@ -27,6 +27,24 @@ forEach(PROTOCOLS).describe('protocol: %s', (protocol: (typeof PROTOCOLS)[number
     await circomkit.compile(CIRCUIT_NAME);
   });
 
+  it('should compile the circuits and not generate the c witness calculator files', async () => {
+    const outPath = await circomkit.compile(CIRCUIT_NAME);
+    expect(existsSync(`${outPath}/${CIRCUIT_NAME}_cpp`)).to.be.false;
+  });
+
+  it('should compile circuit and generate the c witness calculator files', async () => {
+    const circomKitCWitness = new Circomkit({
+      protocol,
+      verbose: false,
+      logLevel: 'silent',
+      cWitness: true,
+    });
+    const outPath = await circomKitCWitness.compile(CIRCUIT_NAME);
+    expect(existsSync(`${outPath}/${CIRCUIT_NAME}_cpp`)).to.be.true;
+    // remove it to prevent the next proof system run to fail
+    rmSync(`${outPath}/${CIRCUIT_NAME}_cpp`, {recursive: true});
```

Comment:
@erhant could it be that this fix #45 btw? saw that the second time it run it started failing until I added this.

Reply:
I'm actually very confused now (bit past my usual bedtime 😆), maybe my fork main branch was a bit outdated, and this was already in `circomkit.test.ts`, and just now saw that in the main repo it was moved to `config.test.ts`. Lmk what you want me to do here, happy to remove it, though I think it might just work to leave it here as long as we remove the `cpp` folder

Reply:
Ah this is a nice find, I think we can leave this here 🙏🏻 I will mention it in the issue, but I think I want to write the test for this specifically and then close it like that.

Reply:
Sounds good, will leave as is then!